infra/terraform/modules/repositories/main.tf

```diff
@@ -103,5 +103,5 @@ resource "github_team_repository" "ci_teams" {
   }
   repository = each.value.repo
   team_id    = each.value.team
-  permission = "triage"
+  permission = "write"
```

Comment:
Although they have write IIUC they won't be able to push any changes directly due to branch protection?

Reply:
Correct - CODEOWNERS for the main branch, which `write` can't override.